### DISCUSSION TITLE
Let Manifest only contain fields needed by ElectionGuard

### DISCUSSION THREAD

**Discussion by JohnLCaron** [microsoft/electionguard#247] [Ideas]
It seems likely that election voting vendors will need a lot of information that electionguard doesn't need. Theres no point in trying to capture it in the Manifest, it will be endless revisions and never actually correct.

So we could simplify Manifest to be just the fields needed by electionguard library. The vendor will have to map their schema to ours, and by keeping ours as simple and clear as possible, it will make their job easier.

It looks to me we could eliminate AnnotatedString, ContactInformation, InternationalizedText, Language, and Party. Also ElectionType and ReportingUnitType. 

A deeper cut might eliminate BallotStyle, Candidate, and GeopoliticalUnit. 

It will be the vendor's job to make a consistent mapping to our simplified Manifest. We can add just the consistency checks needed for the correct functioning of electionguard.

**Comment by rc-ms** [comment]
Hi @JohnLCaron thanks for the suggestions.  Given that the manifest is not exclusively for ElectionGuard but the election vendors themselves as well as other actors (election administrators, ballot printers, etc.), it's not ours to exclusively determine and as such I'm inclined to let it the elements be expansive and self-documenting, as the current version allows. There does need to be a definitive election manifest in both cases. We also don't want to place the vendors in the position of them having to support us; by supporting the NIST standard we present an interoperable stance.

In addition, fields like BallotStyle and GeoPoliticalUnit are critical assignment parameters for determining which ballots voters receive as well as which validations and contests are "in effect" for that voter; those must be preserved. In addition, ElectionType will be important when we need to support primaries, which often have different rules for display and selection.

I share your concerns that the fields be properly documented and supported, however, and will endeavor to provide additional clairty.
